### ISSUE TITLE
[Bug][Grid View][a11y] Save to collection button isn't read properly

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#16416]
### Prerequisites
Have three open tabs.
Talkback should be enabled
Grid view in Tabs Tray should be enabled

### Steps to reproduce
1. Open The Tabs Tray
2. Tap the "Save to collection" button

### Expected behavior
Should read "Save to collection, Button"

### Actual behavior
"Save to collection, Button, 3 of 3"

### Device information

* Android device: Google Pixel 3a (Android 11)
* Fenix version: Nightly Build 201106

### Notes
► Screenshot
![rsz_screenshot_20201106-115052](https://user-images.githubusercontent.com/51314259/98354819-e9037d00-2029-11eb-80fa-948bd9746eab.png)

**Comment by sflorean** [comment]
Verified as fixed on Nightly 11/17 with Pixel 3 (Android 11); "Select tabs, Button, out of grid, double-tap to activate" is read by TalkBack when selecting the button.
